typings/sharify.d.ts

```diff
@@ -16,6 +16,10 @@ declare module "sharify" {
      */
     export interface GlobalData {
       readonly METAPHYSICS_ENDPOINT: string
+      readonly CURRENT_USER?: {
+        id: string
+        accessToken: string
+      }
```

Comment:
This doesn’t seem right, because this will make it possible again to do the wrong thing in SSR code. Is this meant for client-side code?

Reply:
I'll remove this for now and type `(sd as any)` in the client code, and add a fixme for the next PR.